src/js/ListItem.js

```diff
@@ -35,26 +16,33 @@ class ListItem extends Ls {
   };
 
   edit = (e) => {
+    // const list = this.getFromLS()
     const numId = e.currentTarget.id.replace('task-', '');
 
     const dragIco = e.currentTarget.querySelector('.dragndrop-ico');
     const trashIco = e.currentTarget.querySelector('.trash-ico');
+    this.list = this.getFromLS();
 
     trashIco.currentParentEl = e.currentTarget;
     trashIco.currentParentId = numId;
-    trashIco.addEventListener('click', this.removeItem);
-
+    trashIco.addEventListener('click', (e) => {
+      this.RemoveItems.removeItem(e.target);
+      // this.RemoveItems.recalculateIndexes();
```

Comment:
- [OPTIONAL] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable. To make sure that your codebase is always clean, always remember to remove commented-codes.